### DISCUSSION TITLE
502 Bad Gateway - after run yarn setup & try to open ui in browser

### DISCUSSION THREAD

**Discussion by UmangZalariya01** [Budibase/budibase#12557] [Contributing]
![image](https://github.com/Budibase/budibase/assets/146698542/2fb69b0b-87d5-4ec0-bdaa-02cf8c49a567)

I cloned Budibase repo in local & follow steps from docs/CONTRIBUTING.md

please tell me problem is where And anything is missing ?

**Comment by melohagan** [comment]
Hey @UmangZalariya01 

Try the following:

`yarn clean` (you may also need to delete the top-level node_modules folder
`yarn`
`yarn dev`

**Reply by UmangZalariya01** [reply]
Thank You for quick response...👍

I tried this three command ... again same issue 502 bad gateway
anything is missing from my end?

**Reply by melohagan** [reply]
There was a recent change made to include the **account-portal** as a git submodule of the mono repo. It's possible that this problem is related. I'll ask around and see.

**Reply by UmangZalariya01** [reply]
Yess.. @melohagan 
Currently not able to access two sub modules in Packages.

**Packages**
 - account-portal
 - pro 
 
 If you find anything Pls.. revert back.

**Comment by thongbk83** [comment]
Hi all , 
Any update for this topic , I got the same error when try http://localhost:10000/builder/admin . pull source code and follow steps  from docs/CONTRIBUTING.md

**Comment by pengwenan** [comment]
I have also encountered the same problem, how can I solve it
